docs/styling.md

```diff
@@ -0,0 +1,119 @@
+# Styling in Store Framework Jamstack
+
```

Comment:
## Style with Gatsby themes

Gatsby themes are plugins that include a `gatsby-config.js` file and add pre-configured functionality, data sourcing, and/or UI code to Gatsby sites. This approach allows you to control the site styling through a decoupled package, with version control and autonomy for updates. It is also possible to have more than one theme!